packages/vscode-extension/lib/inspector_bridge.js

```diff
@@ -0,0 +1,25 @@
+const agent = globalThis.__radon_agent;
+
+if (!agent) {
+  throw new Error("Radon inspector bridge agent is not installed");
+}
+
+const messageListeners = [];
+
+const inspectorBridge = {
```

Comment:
For me, the term "Inspector" has a specific meaning, relating to the Element Inspector feature. I'd avoid reusing that name for other things -- it's surprising to me that other tools use an "Inspector Bridge", and not just the "Inspector".
Since this is meant to be used for communicating with the extension, I'd rename this to something like `ExtensionBridge`